web/src/js/components/ContentView.jsx

```diff
@@ -33,10 +33,15 @@ function ContentView(props) {
         return <MetaViews.ContentTooLarge {...props} onClick={displayLarge}/>
     }
 
-    const View = ContentViews[contentView] || ContentViews['ViewServer']
+    let view;
+    if(contentView === "Edit") {
+        view = <Edit flow={flow} message={message} onChange={onContentChange}/>
+    } else {
+        view = <ViewServer flow={flow} message={message} contentView={contentView}/>
+    }
```

Comment:
This is more explicit and makes the introspection in ContentLoader.jsx below (which was broken and triggered me looking into this) superfluous.